static/sass/_patterns_navigation.scss

```diff
@@ -1,30 +1,253 @@
 @mixin local-p-navigation {
-  $color-ubuntu-orange: #e95420;
+  $color-ubuntu-orange: #d54110;
 
   .p-navigation {
-    @extend %vf-has-box-shadow;
-
-    font-weight: 400;
-    font-size: 1.125rem;
+    flex-direction: column;
     position: sticky;
-    top: -1px;
+    top: 0;
 
-    .p-navigation__link > a {
-      height: 100%;
+    &.is-sticky {
+      @media screen and (min-width: $breakpoint-medium) {
+        box-shadow: 0 1px 5px 1px rgba(51, 51, 51, 0.2);
+
+        .p-navigation__row:first-child {
+          .p-navigation__nav {
+            &::after {
+              content: none;
+            }
+          }
+        }
+
+        .p-navigation__row:nth-child(2) {
+          .p-navigation__items {
+            position: absolute;
+            top: 0;
+            background-color: $color-x-light;
+          }
+        }
+      }
     }
   }
 
   .p-navigation__banner {
-    background-color: $color-ubuntu-orange;
-    padding-inline-start: 1rem;
+    @media (max-width: $breakpoint-medium - 1px) {
+      padding-inline-start: 0;
+    }
+  }
+
+  .p-navigation__row:first-child {
+    @media (max-width: $breakpoint-medium - 1px) {
+      &::after {
+        background-color: #ccc;
+        content: "";
+        height: 1px;
+        left: 4.5rem;
+        position: absolute;
+        right: 0;
+        top: 55px;
+      }
+    }
+
+    .p-navigation__nav {
+      margin-inline-start: 1rem;
 
-    @media screen and (min-width: 876px) {
-      margin-inline-start: 0.5rem;
+      @media (min-width: $breakpoint-medium) {
+        margin-inline-start: 1.5rem;
+        position: relative;
+
+        &::after {
+          background-color: $color-mid-x-light;
+          bottom: 0;
+          content: "";
+          height: 1px;
+          left: 0;
+          position: absolute;
+          right: 0;
+        }
+      }
+
+      @media (max-width: $breakpoint-medium - 1px) {
+        display: flex;
+        margin-inline-start: 0;
+        position: absolute;
+        left: 4.5rem;
+        top: 0;
+
+        .p-navigation__items.is-website {
+          display: flex;
+        }
+      }
     }
   }
 
-  .p-side-navigation {
-    position: sticky;
-    top: 3rem;
+  .p-navigation__row:nth-child(2) .p-navigation__nav {
+    @media (max-width: $breakpoint-medium - 1px) {
+      &::after {
+        background-color: $color-mid-x-light;
+        content: "";
+        height: 1px;
+        left: 0;
+        position: absolute;
+        right: 0;
+        bottom: 0;
+      }
+
+      .p-navigation__item {
+        .p-subnav__items {
+          hr {
+            margin: 0.75rem 0 0.75rem 1.5rem;
+          }
+        }
+      }
+    }
+
+    .p-navigation__item:first-child .p-navigation__link::before {
+      content: none;
+    }
+
+    @media (min-width: $breakpoint-medium) {
+      justify-content: flex-end;
+    }
+  }
+
+  .p-subnav::after {
+    top: 1.25rem;
```

Comment:
Can you file an issue on Vanilla for this modification. The icon should be central to the menu item.

Reply:
in vanilla it is central, but the height of the navigation in vanilla is 48px, while in this nav I have 56px.

Reply:
I know, I am saying this seems like a bug in Vanilla

Reply:
I'll open a bug, thanks

Reply:
https://github.com/canonical-web-and-design/vanilla-framework/issues/3711